src/pages/IssueCollection/components/IssueCollectionForm.tsx

```diff
@@ -0,0 +1,156 @@
+                id={sft}
+                name={IssueCollectionFieldsEnum.tokenType}
+                onChange={formik.handleChange}
+                type='radio'
+                value={sft}
+              />
+              <label htmlFor={sft} className='text-sm'>
+                {sft}
+              </label>
+            </div>
+          </div>
+        </div>
+        <div className='flex flex-col'>
+          <label
+            htmlFor={IssueCollectionFieldsEnum.tokenName}
+            className='block text-sm font-bold mb-2'
+          >
+            Collection name:
+          </label>
+          <input
+            className={classNames(
+              'block w-full p-2 text-sm text-gray-700 placeholder-gray-400 border border-gray-300 rounded',
+              {
+                'border-red-600':
+                  formik.touched[IssueCollectionFieldsEnum.tokenName] &&
+                  formik.errors[IssueCollectionFieldsEnum.tokenName]
+              }
+            )}
+            data-testid={DataTestIdsEnum.tokenNameInput}
+            id={IssueCollectionFieldsEnum.tokenName}
+            name={IssueCollectionFieldsEnum.tokenName}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            placeholder='Enter token name'
+            value={formik.values[IssueCollectionFieldsEnum.tokenName]}
+          />
+          {formik.touched[IssueCollectionFieldsEnum.tokenName] &&
+            formik.errors[IssueCollectionFieldsEnum.tokenName] && (
+              <div
+                className='text-red-600 text-sm mt-1'
+                data-testid={DataTestIdsEnum.tokenNameError}
+              >
+                {formik.errors[IssueCollectionFieldsEnum.tokenName]}
+              </div>
+            )}
+        </div>
+        <div className='flex flex-col'>
+          <label
+            htmlFor={IssueCollectionFieldsEnum.tokenTicker}
+            className='block text-sm font-bold mb-2'
+          >
+            Collection ticker:
+          </label>
+          <input
+            className={classNames(
+              'block w-full p-2 text-sm text-gray-700 placeholder-gray-400 border border-gray-300 rounded',
+              {
+                'border-red-600':
+                  formik.touched[IssueCollectionFieldsEnum.tokenTicker] &&
+                  formik.errors[IssueCollectionFieldsEnum.tokenTicker]
+              }
+            )}
+            data-testid={DataTestIdsEnum.tokenTickerInput}
+            id={IssueCollectionFieldsEnum.tokenTicker}
+            name={IssueCollectionFieldsEnum.tokenTicker}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            placeholder='Enter token ticker'
+            value={formik.values[IssueCollectionFieldsEnum.tokenTicker]}
+          />
+          {formik.touched[IssueCollectionFieldsEnum.tokenTicker] &&
+            formik.errors[IssueCollectionFieldsEnum.tokenTicker] && (
```

Comment:
There could be variables declared for these long checks, which are also repetitive. 👍